### PR TITLE
Finished results section, re-created renv because we removed a package

### PR DESCRIPTION
**Pull request by derekrodgers**

### CODE REVIEW

File: vancouver-tree-height-geo.Rmd [hunk 1]
```diff
@@ -120,57 +115,64 @@ heatmap_data <- cont_table |>
...
-Because we are dealing with two categorical variables each with multiple levels, a Chi-squared test of Independence/Homogeneity is appropriate
+Because we are dealing with two categorical variables, each with multiple levels, a Chi-squared test of Independence/Homogeneity is appropriate.
 
 The Chi-squared test makes the following assumptions:
 
 1.  The observations are independent.
-2.  The expected counts are sufficiently large (greater than 5 is the typical standard).
+2.  The expected counts are large enough (greater than 5 is typical).
+
+First, the height measurement of a particular tree does not depend on that of another, so we can assume independence. Second almost every intersection of the contingency table has counts much larger than 5, with many in the hundreds or thousands.
```
Comment (ZhengHe-007) [inline vancouver-tree-height-geo.Rmd:159]: You can add a comma behind the word "Second" to improve readability.

File: vancouver-tree-height-geo.Rmd [hunk 2]
```diff
@@ -62,55 +62,50 @@ str(trees) # maybe we don't use this
...
-NOTE TO SELF: add figure captions and for the one below, refer to the tibble above
+The new `trees_subset` dataframe contains one row per tree, with only the columns `NEIGHBOURHOOD_NAME` and `HEIGHT_RANGE_ID`. We can use this along with the `tabyl()` function from the `janitor` package to generate a contingency table. We will then print the contingency table using `kable()`.
 
 ```{r}
 cont_table <- trees_subset |> 
   tabyl(NEIGHBOURHOOD_NAME, HEIGHT_RANGE_ID)
 
 cont_table |> 
-  kable()
+  kable(caption = "Contingency table showing counts of trees in various levels of the tree height variable per levels of the neighbourhood variable. For a mapping of tree height category names, refer to Table 1.")
```
Comment (jessiezhang24) [inline vancouver-tree-height-geo.Rmd:105]: maybe better to highlight where the Table 1 is

**Review by jessiezhang24 — CHANGES_REQUESTED** [review]

File: vancouver-tree-height-geo.Rmd (hunk 1 above)
Comment (derekrodgers) [reply vancouver-tree-height-geo.Rmd:159]: Done

File: vancouver-tree-height-geo.Rmd (hunk 2 above)
Comment (derekrodgers) [reply vancouver-tree-height-geo.Rmd:105]: The kable() function actually labels all the tables sequentually when we knit the PDF, so it all looks correct the way it is when we knit. Good catch, though.

**Pull request merged**